apps/entry/filter_set.py

```diff
@@ -213,6 +219,12 @@ def authoring_organization_types_filter(self, qs, name, value):
             return qs.filter(organization_types__in=[ot.id for ot in value])
         return qs
 
+    def entry_ids_filter(self, qs, name, value):
+        value = eval(value)
```

Comment:
Let's not use `eval` due to security reasons.

`django_filters.ModelMultipleChoiceFilter` should be fine.